docs/Operators.md

```diff
@@ -7475,27 +7475,28 @@ This version of the operator has been available since version 10 of the default
 #### Examples
 
 <details>
-<summary>float_mixed_sign</summary>
+<summary>fmod_mixed_sign</summary>
```

Comment:
>fmod_mixed_sign [](start = 9, length = 15)

You put the name here w/o trailing _example but the actual tests names below has it.